SECURITY.md

```diff
@@ -0,0 +1,34 @@
+# Security policy
+
+Thanks for helping make `setup-beam` safer for everyone.
+
+Find below updated information on our security policy.
+
+## Security
+
+We take the security of this software seriously.
+
+We don't implement a bug bounty program or bounty rewards, but will work with
+you closed to ensure that your findings gets the appropriate handling.
+
+## Reporting Security Issues
+
+If you believe you have found a security vulnerability in this repository,
+please report it to <bryan.paxton@erlef.org>.
```

Comment:
@starbelly, I believe I saw your name associated to this repository's security somewhere, which is why I tentatively chose you here. Lemme know if this is wrong. Thanks.

Reply:
Works for me 👍